### ISSUE TITLE
0.5.0 artifacts missing from Maven Central

### ISSUE THREAD

**Issue by joffrey-bion** [dnault/therapi-runtime-javadoc#16]
It looks like 0.5.0 were not synced between JCenter and Maven Central, because I can only access 0.4.0 on Maven Central, while JCenter has the last version.

It looks like your bintray plugin config in Gradle is set up with disabled sync to Maven Central. It would probably make things easier to deploy if you just activated the sync from there.

**Comment by dnault** [comment]
Really? That's strange. The sync from JCenter was successful, and the artifacts seem to be present in the Maven Central repo:

https://search.maven.org/#search%7Cga%7C1%7Ctherapi-runtime-javadoc

http://repo.maven.apache.org/maven2/com/github/therapi/therapi-runtime-javadoc/0.5.0/
http://repo.maven.apache.org/maven2/com/github/therapi/therapi-runtime-javadoc-scribe/0.5.0/

They haven't been mirrored to mvnrepository.com yet though... that might take a while.

**Comment by dnault** [comment]
That's a good tip about enabling the sync feature in the bintray plugin, thanks.

**Comment by joffrey-bion** [comment]
Interesting, I guess when I first tried to pull the artifacts, it might have been too early, hence the error I got from fetching the dependency.
Then, I was checking the presence of the 0.5.0 there:
https://mvnrepository.com/artifact/com.github.therapi/therapi-runtime-javadoc

But I didn't attempt any more downloads from Central. I confirm the artifacts are there, but are still not indexed by mvnrepository.
